__tests__/app.test.js

```diff
@@ -37,4 +37,17 @@ describe("Get /api/categories", () => {
         expect(response.body.categories).toEqual(result);
       });
   });
+
+  describe("3.5 GET /api", () => {
+    test("", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+          console.log(Object.keys(response.body))
+          expect(Object.keys(response.body).includes('GET /api')).toBe(true)
```

Comment:
This test will not be robust enough when you add to the endpoints. If you require the endpoints.json into this file, think about how you can compare this to the response to ensure they look the same.

Reply:
It didn't occur to me to require the JSON into the test file and so I was checking for the 1 thing that will always be consistent. I will now change it to check against the file required into test vs the file required into the controller